bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/SatelBinding.java

```diff
@@ -248,4 +247,68 @@ private synchronized boolean switchForceRefresh(boolean forceRefresh) {
         this.forceRefresh = forceRefresh;
         return oldValue;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean isConnected() {
+        logger.debug("[SatelCommModule] isConnected");
+        return this.satelModule != null && this.satelModule.isConnected();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean sendCommand(SatelCommand command) {
+        logger.debug("[SatelCommModule] sendCommand");
+        if (this.satelModule == null) {
+            return false;
+        }
+        if (!this.satelModule.sendCommand(command, true)) {
+            return false;
+        }
+
+        while (!Thread.interrupted()) {
+            // wait for command state change
+            try {
+                synchronized (command) {
+                    command.wait(this.satelModule.getTimeout());
+                }
+            } catch (InterruptedException e) {
+                // ignore, we just leave the loop
```

Comment:
The empty `catch` block is followed by the `switch` statement below, so the `while` loop isn't really left until more instructions are performed.  So it's confusing, even if it works correctly in practice.